RadixWallet/Clients/IOSSecurityClient/IOSSecurityClient+Live.swift

```diff
@@ -0,0 +1,9 @@
+import IOSSecuritySuite
+
+extension IOSSecurityClient: DependencyKey {
+	public static let liveValue = Self(
+		isJailbroken: {
+			IOSSecuritySuite.amIJailbroken() || true
```

Comment:
do we really wanna default to `true` ? feels a bit to aggresive?

Reply:
This is even more aggressive, it's an OR! Forgot to remove it, thanks for spotting it. Btw I first put ?? there, confusing them the other way around.

Reply:
Yeah haha I missed that it was OR 😅😅😅